docs/releases/1.11.txt

```diff
@@ -133,6 +133,7 @@ Database backends
 Email
 ~~~~~
 
+* Added a new :setting:`EMAIL_USE_LOCALTIME` to send SMTP date headers in local time zone.
 * ...
```

Comment:
Remove this and add the setting to the "Core Settings Topical Index" at the bottom of this page.